HBASE.md

```diff
@@ -131,43 +131,31 @@ put 'sbr_control_db:unit_links' , '38576395~PAYE~201801', 'l:p_LEU', '1928374659
-put 'sbr_control_db:legal_unit', '3210000001~201801~1000012345000002', 'd:tradingstyle', 'A'
-put 'sbr_control_db:legal_unit', '3210000001~201801~1000012345000002', 'd:address1', 'address1-123'
-put 'sbr_control_db:legal_unit', '3210000001~201801~1000012345000002', 'd:address2', 'address2-123'
-put 'sbr_control_db:legal_unit', '3210000001~201801~1000012345000002', 'd:address3', 'address3-123'
-put 'sbr_control_db:legal_unit', '3210000001~201801~1000012345000002', 'd:address4', 'address4-123'
-put 'sbr_control_db:legal_unit', '3210000001~201801~1000012345000002', 'd:address5', 'address5-123'
-put 'sbr_control_db:legal_unit', '3210000001~201801~1000012345000002', 'd:postcode', 'postcode-123'
-put 'sbr_control_db:legal_unit', '3210000001~201801~1000012345000002', 'd:sic07', 'sic07-123'
-put 'sbr_control_db:legal_unit', '3210000001~201801~1000012345000002', 'd:jobs', '123'
-put 'sbr_control_db:legal_unit', '3210000001~201801~1000012345000002', 'd:turnover', '10'
-put 'sbr_control_db:legal_unit', '3210000001~201801~1000012345000002', 'd:legalStatus', 'legalStatus-123'
-put 'sbr_control_db:legal_unit', '3210000001~201801~1000012345000002', 'd:tradingStatus', 'legalStatus-123'
-
-put 'sbr_control_db:legal_unit', '3210000001~201801~1000012345000001', 'd:UBRN', '1000012345000001'
-put 'sbr_control_db:legal_unit', '3210000001~201801~1000012345000001', 'd:ern', '3210000001'
-put 'sbr_control_db:legal_unit', '3210000001~201801~1000012345000001', 'd:entref', 'entref-123'
-put 'sbr_control_db:legal_unit', '3210000001~201801~1000012345000001', 'd:crn', 'crn-123'
-put 'sbr_control_db:legal_unit', '3210000001~201801~1000012345000001', 'd:name', 'Company X'
-put 'sbr_control_db:legal_unit', '3210000001~201801~1000012345000001', 'd:tradingstyle', 'A'
-put 'sbr_control_db:legal_unit', '3210000001~201801~1000012345000001', 'd:address1', 'address1-123'
-put 'sbr_control_db:legal_unit', '3210000001~201801~1000012345000001', 'd:address2', 'address2-123'
-put 'sbr_control_db:legal_unit', '3210000001~201801~1000012345000001', 'd:address3', 'address3-123'
-put 'sbr_control_db:legal_unit', '3210000001~201801~1000012345000001', 'd:address4', 'address4-123'
-put 'sbr_control_db:legal_unit', '3210000001~201801~1000012345000001', 'd:address5', 'address5-123'
-put 'sbr_control_db:legal_unit', '3210000001~201801~1000012345000001', 'd:postcode', 'postcode-123'
-put 'sbr_control_db:legal_unit', '3210000001~201801~1000012345000001', 'd:sic07', 'sic07-123'
-put 'sbr_control_db:legal_unit', '3210000001~201801~1000012345000001', 'd:jobs', '123'
-put 'sbr_control_db:legal_unit', '3210000001~201801~1000012345000001', 'd:turnover', '10'
-put 'sbr_control_db:legal_unit', '3210000001~201801~1000012345000001', 'd:legalStatus', 'legalStatus-123'
-put 'sbr_control_db:legal_unit', '3210000001~201801~1000012345000001', 'd:tradingStatus', 'legalStatus-123'
-
-
+put 'sbr_control_db:legal_unit', '3210000001~201801~1000012345000002', 'd:ubrn', '1000012345000002'
```

Comment:
Should we be using the actual LEU test data here? (Not that it really matters)

Reply:
For my own selfish purposes, I have found the json to be easier to scan with obviously fake data.  For example, it is easier to check that birth_date & death_date are being assigned correctly (and not mixed-up) if they contain "birth-date" or "death-date" rather than actual date values.

If our aim is to use this file for anything other than these quick manual tests, then yes, we probably should switch to the sample datasets.

Reply:
That sounds fair enough. I'd say we only need to consider using real values where they are transformed in the user interface, e.g. an employment band of `A` will be converted to `1-100` etc.